eox_tagging/api/v1/test/test_viewset.py

```diff
@@ -342,6 +344,30 @@ def test_filter_by_type(self, _):
 
         self.assertEqual(response.status_code, 200)
 
+    @patch_permissions
+    def test_filter_using_after_datetime(self, _):
+        query_params = {
+            "activation_date_0": "2020-10-10 10:20:30",
+        }
+
+        response = self.client.get(self.URL, query_params)
+
+        results = response.json().get("results")
+        datetimes = [tag.get("activation_date") > "2020-10-10T10:20:30Z" for tag in results]
```

Comment:
Are you comparing strings ?

Reply:
... no

Reply:
I changed it, now I compare datetime objects @andrey-canon